README.md

```diff
@@ -14,3 +14,22 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+1. Asymptotic analysis could be misleading compared to practice in a few different ways. 
+
+    a) For small input sizes, complexity can be misleading because between two input classes, say $\Theta\left(n\right)$ and $\Theta\left(log(n)\right)$, it seems that the first will always be slower than the second. However, the reality might be that the first's complexity is $n$ and the second's is $20log_{2}n$ due to the overhead of data structures or processes needed to set up the asymptotically more efficient solution. This is due to the parts of the asymptotic notations definition on constants. Essentially, when we are talking about behaviour as the input size nears infinity, constants aren't going to be important. It won't take terribly long for that overhead to pay off, but if you are only going to be using your algorithm on small input sizes, the asymptotically worse $\Theta\left(n\right)$ may be more efficient to use in practice.
+
+    b) For Big $\Omega$ and Big O bounds specifically, they can be misleading because they don't have to be tight asymptotic bounds. In comparison, Big $\Theta$ is quite tightly asymptoticly bounded. For this reason, Big $\Omega$ and Big O bounds can trip up novice programmers if they don't realize the difference.
+
+    c) A third misleading aspect of asymptotic analysis is that it can't actually be used to measure how long an algorithm will take to run, just how its runtime scales with input size. The reason you can't just plug in an input size and get a time to run is that in practice when the algorithm is run on the machine there are a lot of factors that could affect the runtime in practice, such as how information is cached and what other programs and processes are running in the background of whatever is running the program.
```

Comment:
I don't see how this is misleading -- asymptotic analysis doesn't claim that you can just plug in a number and get the runtime.